<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Login - Hasło - Serwis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="loginEdit">
        <property name="placeholderText">
         <string>Podaj login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit">
        <property name="placeholderText">
         <string>Podaj hasło</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serviceEdit">
        <property name="placeholderText">
         <string>Serwis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="acceptButton">
        <property name="text">
         <string>Zatwierdź</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Usuń</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>Edytuj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>Generuj hasło</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Wyczyść liste</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hideButton">
        <property name="text">
         <string>Ukryj/Wyświetl hasła</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionSaveFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaveFile">
   <property name="text">
    <string>Zapisz do pliku</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Wczytaj z pliku</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>O autorze</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
